rskj-core/src/main/java/co/rsk/peg/BridgeSupport.java

```diff
@@ -2504,6 +2504,13 @@ public long getNextPegoutCreationBlockNumber() {
         return activations.isActive(RSKIP271) ? provider.getNextPegoutHeight().orElse(0L) : 0L;
     }
 
+    public int getQueuedPegoutsCount() {
+        if (activations.isActive(RSKIP271)) {
+            return provider.getReleaseRequestQueueSize();
+        }
+        return 0;
```

Comment:
```suggestion
        return activations.isActive(RSKIP271) ? provider.getReleaseRequestQueueSize() : 0;
```
Just a suggestion, if you like it it's shorter

Reply:
Resolved

Reply:
You removed RSKIP check

Reply:
Resolved